proposals/pr-contrib-guide.md

```diff
@@ -0,0 +1,508 @@
+# Improvements to the AWS CLI Contribution Process
+
+Proposal         | Metadata
+---------------- | -------------
+**Author**       | Kenneth Daily
+**Status**       | Proposed
+**Created**      | 19-October-2021
```

Comment:
Let's update the created date to either today or when we plan to send a PR to the CLI repo.